radis/tools/database.py

```diff
@@ -2372,7 +2372,11 @@ def _load_new_file(self, file, binary=False):
 
         out = s.get_conditions().copy()
 
-        # Add filename, and a link to the Spectrum object itself
+        # Add filename, name and a link to the Spectrum object itself
+        if s.name == None:
+            out["name"] = s.get_name()
```

Comment:
@erwanp when `s.name == None`, the csv actually stores it as the Filename (eg -`20210629_Tvib3000K_Trot3000K.spec`
, I wonder why?